deploy/summit/.env

```diff
@@ -1,11 +1,70 @@
 # INTERACTION BETWEEN SERVICES
 LOVE_PRODUCER_WEBSOCKET_HOST=love-nginx/manager/ws/subscription
+
+# AuxTel
+LOVE_PRODUCER_WEBSOCKET_HOST_ATAOS=love-nginx/manager/auxtel/ws/subscription
+LOVE_PRODUCER_WEBSOCKET_HOST_ATDome=love-nginx/manager/auxtel/ws/subscription
+LOVE_PRODUCER_WEBSOCKET_HOST_ATDomeTrajectory=love-nginx/manager/auxtel/ws/subscription
+LOVE_PRODUCER_WEBSOCKET_HOST_ATHexapod=love-nginx/manager/auxtel/ws/subscription
+LOVE_PRODUCER_WEBSOCKET_HOST_ATMCS=love-nginx/manager/auxtel/ws/subscription
+LOVE_PRODUCER_WEBSOCKET_HOST_ATPneumatics=love-nginx/manager/auxtel/ws/subscription
+LOVE_PRODUCER_WEBSOCKET_HOST_ATPtg=love-nginx/manager/auxtel/ws/subscription
+
+# LATISS
+LOVE_PRODUCER_WEBSOCKET_HOST_ATArchiver=love-nginx/manager/latiss/ws/subscription
+LOVE_PRODUCER_WEBSOCKET_HOST_ATCamera=love-nginx/manager/latiss/ws/subscription
+LOVE_PRODUCER_WEBSOCKET_HOST_ATHeaderService=love-nginx/manager/latiss/ws/subscription
+LOVE_PRODUCER_WEBSOCKET_HOST_ATSpectrograph=love-nginx/manager/latiss/ws/subscription
+LOVE_PRODUCER_WEBSOCKET_HOST_ATOCPS=love-nginx/manager/latiss/ws/subscription
+
+# ComCam
+LOVE_PRODUCER_WEBSOCKET_HOST_CCArchiver=love-nginx/manager/comcam/ws/subscription
+LOVE_PRODUCER_WEBSOCKET_HOST_CCCamera=love-nginx/manager/comcam/ws/subscription
+LOVE_PRODUCER_WEBSOCKET_HOST_CCHeaderService=love-nginx/manager/comcam/ws/subscription
+LOVE_PRODUCER_WEBSOCKET_HOST_CCOCPS=love-nginx/manager/comcam/ws/subscription
+
+# MainTel
+LOVE_PRODUCER_WEBSOCKET_HOST_CameraHexapod=love-nginx/manager/maintel/ws/subscription
+LOVE_PRODUCER_WEBSOCKET_HOST_M2Hexapod=love-nginx/manager/maintel/ws/subscription
+LOVE_PRODUCER_WEBSOCKET_HOST_MTAOS=love-nginx/manager/maintel/ws/subscription
+LOVE_PRODUCER_WEBSOCKET_HOST_MTDome=love-nginx/manager/maintel/ws/subscription
+LOVE_PRODUCER_WEBSOCKET_HOST_MTDomeTrajectory=love-nginx/manager/maintel/ws/subscription
+LOVE_PRODUCER_WEBSOCKET_HOST_MTMount=love-nginx/manager/maintel/ws/subscription
+LOVE_PRODUCER_WEBSOCKET_HOST_MTPtg=love-nginx/manager/maintel/ws/subscription
+LOVE_PRODUCER_WEBSOCKET_HOST_MTRotator=love-nginx/manager/maintel/ws/subscription
+
+# M1M3
+LOVE_PRODUCER_WEBSOCKET_HOST_MTM1M3=love-nginx/manager/m1m3/ws/subscription
+
+# M2
+LOVE_PRODUCER_WEBSOCKET_HOST_MTM2=love-nginx/manager/m2/ws/subscription
+
+# ObsSys
+LOVE_PRODUCER_WEBSOCKET_HOST_ATScriptQueue=love-nginx/manager/obssys/ws/subscription
+LOVE_PRODUCER_WEBSOCKET_HOST_ATScheduler=love-nginx/manager/obssys/ws/subscription
+LOVE_PRODUCER_WEBSOCKET_HOST_MTScheduler=love-nginx/manager/obssys/ws/subscription
+LOVE_PRODUCER_WEBSOCKET_HOST_MTScriptQueue=love-nginx/manager/obssys/ws/subscription
+LOVE_PRODUCER_WEBSOCKET_HOST_LOVE=love-nginx/manager/obssys/ws/subscription
+LOVE_PRODUCER_WEBSOCKET_HOST_Watcher=love-nginx/manager/obssys/ws/subscription
+
+# ESS
```

Comment:
I know the comments are meaningless for the application and merely are there for us. But strictly speaking DIMM1, DIMM2, HVAC, PMD and WeatherStation are not part of ESS. However, all items, including the ESS ones, are part of EAS. I'll leave it for you to decide if you want to modify this.

Reply:
I'll address this in the cycle 23 configuration updates.